vita3k/modules/SceDriverUser/SceAppMgrUser.cpp

```diff
@@ -75,8 +75,17 @@ EXPORT(int, sceAppMgrAddContMount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppMgrAppDataMount) {
-    TRACY_FUNC(sceAppMgrAppDataMount);
+EXPORT(int, sceAppMgrAppDataMount, int id, const char *mount_point) {
+    TRACY_FUNC(sceAppMgrAppDataMount, id, mount_point);
+    switch (id) {
+    case 105:
```

Comment:
Isn't there an enum for all the possible IDs?

Reply:
off course, i have just only tested it

Reply:
done, and reworked, is wrong before